freqtrade/commands/cli_options.py

```diff
@@ -484,6 +484,18 @@ def __init__(self, *args, **kwargs):
         type=float,
         metavar='FLOAT',
     ),
+    "hyperopt_list_min_objective": Arg(
+        '--min-objective',
+        help='Select epochs on above objective.',
+        type=float,
+        metavar='FLOAT',
+    ),
+    "hyperopt_list_max_objective": Arg(
+        '--max-objective',
+        help='Select epochs on below objective.',
```

Comment:
@hroff-1902 Is this wording intentional or was it missed in the other PR's?

It's in line with the other "hyperopt_list_*" options - but it sounds kinda odd to me ... (both for this, and for some other options ...)

wouldn't `"Select epochs below objective."` and `"Select epochs above objective."` sound like ... better english?

Reply:
The wording is wrong, but I basically copied and pasted from other flags, I'll fix them.